### PR TITLE
Update Croatian translations in "data" folder

### PR DESCRIPTION
**Pull request by milotype**

### CODE REVIEW

**Review by kevinpapst — CHANGES_REQUESTED** [review]
Looks good, thanks 👍 

Just one issue:
can you please revert the `url` changes in `_data/trans-menu.yml`. Changing `name` works(!), but the URL should stay the same. Mainly because it is already "out there" and you should not change URLs that Google already knows.

**Comment by milotype** [comment]
Yes, I can change it ... but IMO URL's like `https://www.kimai.org/hr/download.html` (translated into `url: /hr/preuzimanje.html`) and slugs like `https://www.kimai.org/hr/store/` (translated into `https://www.kimai.org/hr/trgovina/` may, and IMO and as far as I know, they can/should be tanslated. Google will learn it ...

Nevertheless, decide what you want, and I will make the changes ... and will follow it for other translations.

**Comment by kevinpapst** [comment]
The website is run by Jekyll, which has terrible/bad support for redirects. And changing the URL has very limited SEO impact. So yes please, change the URLs.

File: _data/trans-menu.yml [hunk 1]
```diff
@@ -236,7 +236,7 @@ hr:
     name: Donacije
     title: Podrži Kimai putem donacije
     icon: fas fa-donate
-    url: /hr/donacije.html
+    url: /hr/donate/
```
Comment (kevinpapst) [inline _data/trans-menu.yml:239]:
```suggestion
    url: /hr/donacije.html
```

File: _data/trans-menu.yml [hunk 2]
```diff
@@ -259,12 +259,12 @@ hr:
   about:
     name: Informacije
     icon: far fa-clock
-    url: /hr/o-programu-kimai.html
+    url: /hr/about/
```
Comment (kevinpapst) [inline _data/trans-menu.yml:262]:
```suggestion
    url: /hr/o-programu-kimai.html
```

File: _data/trans-menu.yml [hunk 3]
```diff
@@ -259,12 +259,12 @@ hr:
...
 
   demo:
     name: Demo
     icon: fas fa-desktop
-    url: /hr/demo.html
+    url: /hr/demo/
```
Comment (kevinpapst) [inline _data/trans-menu.yml:267]:
```suggestion
    url: /hr/demo.html
```

**Review by kevinpapst — CHANGES_REQUESTED** [review]
thanks for reverting, but the new URLs are still not the old ones ;-)

**Comment by milotype** [comment]
OK, I've commited your changes.
But I still don't understand when to use just an URL slug and when to use an URL with a `.html` extension.
I'm really sorry, but as I'm not a programmer and I sometimes do not understand such issues ... :-(

**Comment by kevinpapst** [comment]
Nothing to be sorry about, that is why there are "code reviews". I know Git is much more complicated than using weblate and I really appreciate your support here!!!!

Think of URLs as your computers hard drive. Using `.html` or a slug means it is a file, using a `/` means it is a directory where a search engine will find more files (e.g. the marketplace and all plugins inside).

There is no real difference between adding and removing the .html, but there are technical reasons to better use .html.
And having a directory means that a file called `index.html` is served. Technical nonsense that I do not want to explain in detail here 😁 And at some points I made a mistake in the past, but I do not want to correct that, because changing URLs can lead to some serious consequences in Google.

**Review by kevinpapst — APPROVED** [review]
Thank you @milotype 👍

**Pull request merged**